src/lensProp.js

```diff
@@ -23,7 +25,6 @@ var prop = require('./prop');
  *     phraseLens.set('Ooh Betty', obj1); //=> { phrase: 'Ooh Betty'}
  *     phraseLens.map(R.toUpper, obj2); //=> { phrase: "WHAT'S ALL THIS, THEN?"}
  */
-module.exports = function(k) {
+module.exports = _curry1(function(k) {
```

Comment:
Mind naming this function expression? We're not very consistent, but I know @CrossEye likes named function expressions.